src/SprykerEco/Client/Unzer/Dependency/Client/UnzerToZedRequestClientBridge.php

```diff
@@ -31,7 +31,7 @@ public function __construct($zedRequestClient)
      *
      * @return \Spryker\Shared\Kernel\Transfer\TransferInterface
      */
-    public function call($url, TransferInterface $object, $requestOptions = null)
+    public function call($url, TransferInterface $object, $requestOptions = null): TransferInterface
```

Comment:
```suggestion
    public function call(string $url, TransferInterface $object, ?array $requestOptions = null): TransferInterface
```

Reply:
https://spryker.atlassian.net/wiki/spaces/RFC/pages/3010003045/ACCEPTED+RFC+Consistent+return+type+strictness+for+extension